nucypher/characters/lawful.py

```diff
@@ -993,11 +989,13 @@ def run(
         ritualist: bool = True,
         hendrix: bool = True,
         start_reactor: bool = True,
-        prometheus_config: "PrometheusMetricsConfig" = None,
         preflight: bool = True,
         block_until_ready: bool = True,
         eager: bool = False,
     ) -> None:
+        # Locally scoped to prevent import without prometheus explicitly installed
```

Comment:
If prometheus is installed, this should no longer need to be locally scoped

Reply:
It makes sense, thanks. Done ✅

My only concern is if this could cause any error in Porter. If I understand correctly, adding prometheus_client to the requirements will cause Porter to also install this dependency when deploying.

Some checking should be done before merging this.

Reply:
> My only concern is if this could cause any error in Porter. If I understand correctly, adding prometheus_client to the requirements will cause Porter to also install this dependency when deploying.

Fair. But if the goal is to have prometheus always be installed so that Ursula can always use it, then I'm not sure we can do anything about that.

You are right that while Porter would ultimately install the prometheus dependency (impliclty as part of the `nucypher` dependency), it won't use it - but that may also apply to other existing `nucypher` dependencies anyway.

Reply:
Done!